### DISCUSSION TITLE
Response is 'Sequence is longer than 400.'

### DISCUSSION THREAD

**Discussion by CNDOTA** [facebookresearch/esm#391] [ESM Metagenomic Atlas]
When I input the protein sequences with length larger than 400, I get the information 'Sequence is longer than 400.'.

Can I predict the protein structure with length > 400 with your emsfold API?

Thank you very much!

**Comment by tomsercu** [comment]
We're working on supporting longer sequence lengths;
in the meantime you can use colabfold or huggingface notebook on colab to run this in the browser on a dedicated GPU.
